src/Rel8.hs

```diff
@@ -43,7 +43,6 @@ module Rel8
 
   , Table(..)
   , HTable
-  , Recontextualize
```

Comment:
Do we need to export `Transposes`?

Reply:
We can export it if you want, but the reason I exported `Recontextualize` before was because there were cases where user would need to write instances of `Recontextualize`, whereas now `Transposes` is a glorified type synonym and no instances are necessary.